### PR TITLE
Add FIATLUX_DIS_F{2,L}

### PR DESCRIPTION
**Pull request by niclaurenti**
Adding cards to generate FIATLUX_DIS_F{2,L}

### CODE REVIEW

File: FIATLUX_DIS_FL/metadata.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+nnpdf_id=FIATLUX_DIS_FL
+fktable_id=FIATLUX_DIS_FL
+x1_label=Q2
+x1_label_tex=$Q^2$
+x1_unit=GeV^2
+x2_label=x
+x2_label_tex=$x$
+x2_unit=
+y_label=FLp
+y_label_tex=$F_{2,\mathrm{tot}}^{(\mathrm{p})}$
```
Comment (felixhekhorn) [inline FIATLUX_DIS_FL/metadata.txt:14]:
```suggestion
y_label_tex=$F_{L,\mathrm{tot}}^{(\mathrm{p})}$
```

**Comment by niclaurenti** [comment]
> * should we add the generating script?
> * at minimum we should say somewhere what the grid is spanning, e.g. x=np.geomspace(a,b,c), Q2=np.geomspace(d,e,f)

It's ok for me:
the script I used is

```
import numpy as np
from eko.interpolation import make_grid
import yaml

with open('/Users/niccololaurenti/n3pdf/template_pinefarm.yaml', 'r') as file:
    d=yaml.safe_load(file)

xgrid = make_grid(30, 20, n_high=0, x_min=1e-6).tolist()
q2grid = np.geomspace(1., 1e8, 100).tolist()
list_grid = []

for x in xgrid:
    for q2 in q2grid:
        tmp = {'Q2': q2, 'x': x, 'y': 1.0}
        list_grid.append(tmp)
        
d['observables']['F2_total'] = list_grid

with open('runcard_fiatlux_dis.yaml', 'w+') as ff:
    yaml.dump(d, ff)
```

where the template is 
```
NCPositivityCharge: null
PolarizationDIS: 0.0
ProjectileDIS: electron
PropagatorCorrection: 0.0
TargetDIS: proton
interpolation_is_log: true
interpolation_polynomial_degree: 4
interpolation_xgrid:
- 1.9999999999999954e-07
- 3.034304765867952e-07
- 4.6035014748963906e-07
- 6.984208530700364e-07
- 1.0596094959101024e-06
- 1.607585498470808e-06
- 2.438943292891682e-06
- 3.7002272069854957e-06
- 5.613757716930151e-06
- 8.516806677573355e-06
- 1.292101569074731e-05
- 1.9602505002391748e-05
- 2.97384953722449e-05
- 4.511438394964044e-05
- 6.843744918967896e-05
- 0.00010381172986576898
- 0.00015745605600841445
- 0.00023878782918561914
- 0.00036205449638139736
- 0.0005487795323670796
- 0.0008314068836488144
- 0.0012586797144272762
- 0.0019034634022867384
- 0.0028738675812817515
- 0.004328500638820811
- 0.006496206194633799
- 0.009699159574043398
- 0.014375068581090129
- 0.02108918668378717
- 0.030521584007828916
- 0.04341491741702269
- 0.060480028754447364
- 0.08228122126204893
- 0.10914375746330703
- 0.14112080644440345
- 0.17802566042569432
- 0.2195041265003886
- 0.2651137041582823
- 0.31438740076927585
- 0.3668753186482242
- 0.4221667753589648
- 0.4798989029610255
- 0.5397572337880445
- 0.601472197967335
- 0.6648139482473823
- 0.7295868442414312
- 0.7956242522922756
- 0.8627839323906108
- 0.9309440808717544
- 1
observables:
  F2_total:
  - Q2: 1.
    x: 0.01
    y: 1.
prDIS: EM
```

**Comment by felixhekhorn** [comment]
Okay, then I suggest you add a `README.md` say to the F2 folder with [your post](https://github.com/NNPDF/pinecards/pull/173#issuecomment-1633859200) ... or add a `_script.py` and `_template.yaml`

**Comment by niclaurenti** [comment]
> Okay, then I suggest you add a `README.md` say to the F2 folder with [your post](https://github.com/NNPDF/pinecards/pull/173#issuecomment-1633859200) ... or add a `_script.py` and `_template.yaml`

I added both the F2 and FL scripts to prevent people from copy pasting without remembering that they have to change a character

**Comment by niclaurenti** [comment]
@felixhekhorn Now this is complete. Whenever you want we can merge

**Review by felixhekhorn — APPROVED** [review]
Looks good to me

**Pull request merged**